src/test/java/de/bonndan/nivio/assessment/StatusValueTest.java

```diff
@@ -0,0 +1,28 @@
+package de.bonndan.nivio.assessment;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StatusValueTest {
+
+    @Test
+    void highestOf() {
+        List<StatusValue> statusValueSet = new ArrayList<>();
+        statusValueSet.add(new StatusValue("foo", Status.GREEN));
+        statusValueSet.add(new StatusValue("bar", Status.ORANGE));
+        statusValueSet.add(new StatusValue("baz", Status.ORANGE));
+
+        List<StatusValue> highest = StatusValue.highestOf(statusValueSet);
```

Comment:
Did not notice that before, but I would expect ``highestOf()`` to return a single item (or an ``Optional`` of an item).

Reply:
No, for instance a landscape item could have Status.RED concerning security AND health, and neither of them is higher than the other. The method is unused yet, however.